src/server/game/AI/SmartScripts/SmartScriptMgr.h

```diff
@@ -167,7 +167,15 @@ enum SMART_EVENT
     SMART_EVENT_DISTANCE_GAMEOBJECT      = 76,      // guid, entry, distance, repeat
     SMART_EVENT_COUNTER_SET              = 77,      // id, value, cooldownMin, cooldownMax
 
-    SMART_EVENT_END                      = 78
+    SMART_EVENT_TC_END                   = 78,
+
+    /* AC Custom Events */
+    SMART_EVENT_AC_START                 = 100,
+
+    SMART_EVENT_NEAR_PLAYERS             = 101,      // min, radius, first timer, check timer
+    SMART_EVENT_NEAR_PLAYERS_NEGATION    = 102,      // min, radius, first timer, check timer
+
+    SMART_EVENT_AC_END                   = 103
```

Comment:
if it's not hard for you, shall we make it `2xx` instead of `1xx` ? for consistency